db-scripts/src/main/resources/cgds.sql

```diff
@@ -774,10 +789,25 @@ CREATE TABLE `clinical_event_data` (
   FOREIGN KEY (`CLINICAL_EVENT_ID`) REFERENCES `clinical_event` (`CLINICAL_EVENT_ID`) ON DELETE CASCADE
 );
 
+-- --------------------------------------------------------
+CREATE TABLE `reference_genome_gene` (
+    `ENTREZ_GENE_ID` int(11) NOT NULL,
+    `REFERENCE_GENOME_ID` int(4) NOT NULL,
+    `CHR` varchar(4) DEFAULT NULL,
+    `CYTOBAND` varchar(64) DEFAULT NULL,
+    `EXONIC_LENGTH` int(11) DEFAULT NULL,
+    `START` bigint(20) DEFAULT NULL,
+    `END` bigint(20) DEFAULT NULL,
+    `ENSEMBL_GENE_ID` varchar(64) DEFAULT NULL,
```

Comment:
what is the use case here? Do we really want to have such mapping here? Is it even 1 to 1 mapping? And if yes, then a unique key for `(ENSEMBL_GENE_ID, REFERENCE_GENOME_ID)` should also be added right? 
@n1zea144 what are your thoughts on this?

Reply:
Hi @pieterlukasse @n1zea144 , Trevor wants to have ** ENSEMBL_GENE_ID** tracked for a easy search, since it is more widely used than **ENTREZ_GENE_ID**. Unfortunately, we can not just  replace **ENTREZ_GENE_ID** with **ENSEMBL_GENE_ID** due to the great impact on the source code. Hope in the future we could revisit here. For now, we just make it an optional key.

Reply:
@khzhu I don't think we should include ensembl support this way. Could you please remove this field? We have some plan to support search of ensembl IDs in the frontend if that's what Trevor is interesed in.